.github/workflows/scripts/readme.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/zsh -ex
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+dirs=(`find . -type f -name "go.mod" -not -path "*/internal/*" -exec dirname {} \; | sort | egrep  '^./'`)
```

Comment:
How about adding a mention in https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#documentation that each Go modules should have a `README.md` with a title and PkgGoDev badge?